docs/howtos/early_stopping.rst

```diff
@@ -0,0 +1,124 @@
+Early Stopping
+=============================
+Early stopping is used while building model to stop training loop when specific conditions are met. Some such conditions are:
```

Comment:
Can you please just copy/paste a definition of early stopping? This sentence is a bit hard to parse.